pkg/syncer/syncer.go

```diff
@@ -176,7 +176,7 @@ func getAllGVRs(discoveryClient discovery.DiscoveryInterface, resourcesToSync ..
 			return nil, err
 		}
 	}
-	gvrstrs := []string{"namespaces.v1."} // A syncer should always watch namespaces
+	gvrstrs := []string{"namespaces.v1.", "serviceaccounts.v1.", "configmaps.v1.", "secrets.v1."} // A syncer should always watch namespaces, serviceaccounts, secrets and configmaps.
```

Comment:
does this also mean they are synced by default?

Reply:
Yes... But we can tune the scheduler to avoid that.

Reply:
@sttts I've pushed a WIP commit, trying to solve that issue, let me know what you think

Reply:
https://github.com/kcp-dev/kcp/pull/680